src/main/java/capstone/mfslbackend/repository/FantasyTeamRepository.java

```diff
@@ -3,5 +3,10 @@
 import capstone.mfslbackend.model.FantasyTeam;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.List;
+
 public interface FantasyTeamRepository extends JpaRepository<FantasyTeam, Long> {
+    List<FantasyTeam> findFantasyTeamsByFantasyLeagueId(Long leagueId);
+
+    FantasyTeam findFantasyTeamById(Long homeTeamId);
```

Comment:
Don't use this. Its important that it returns an Optional and that it forces you to check that the fantasyTeam actually exists to prevent null pointer errors. Use FantasyTeamRespository.findById()